devtools/parcelmanifest/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	texttemplate "text/template"
+)
+
+var tpl = `package web
+
+var parcelAssetMap map[string]string
+
+func init() {
+	parcelAssetMap = {{ $.AssetMap }}
+}
+
+func GetHashedName(file string) string {
+	return parcelAssetMap[file][1:]
+}
+
+`
```

Comment:
Ultimately we should have 2 files. `parcel.go` and `parcel_gen.go`.

In `parcel.go`, 

```golang
package web

var parcelAssetMap map[string]string

func GetHashedName(file string) string {
  n := parcelAssetMap[file]
  if n == "" {
    return file
  }
  return n
}
```